docs/data-sources/packetfabric_locations_cloud.md

```diff
@@ -0,0 +1,73 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "packetfabric_locations_cloud Data Source - packetfabric"
+subcategory: ""
+description: |-
+  
+---
+
+# packetfabric_locations_cloud (Data Source)
+
+
+```terraform
+data "packetfabric_locations_cloud" "cloud_location_1" {
+  provider = packetfabric
+  cloud_provider = "aws"
+  cloud_connection_type = "hosted"
+}
+
+output "packetfabric_locations_cloud" {
+  value = data.packetfabric_locations_cloud.cloud_location_1  
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cloud_connection_type` (String) Filter locations by cloud connection type.
```

Comment:
@elXaptation Could we add the available options: ``aws, azure, packet, google, ibm, oracle, salesforce, webex`` as per [API spec](https://docs.packetfabric.com/api/v2/swagger/#/Locations/get_cloud_location_list).